index.html

```diff
@@ -4,7 +4,6 @@
     <meta charset="utf-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width,initial-scale=1.0" />
-    <link rel="icon" href="favicon.svg" type="image/svg+xml">
```

Comment:
let's keep the icon and fix that instead of removing it entirely

Reply:
This removed line is a duplicate "link rel="icon"" entry .

The correct icon entry is already in line 13, which you don't see in the small snippet.

Reply:
Ah my mistake - I didnt spot that indeed!